app/src/main/java/com/hedvig/app/feature/insurance/ui/InsuranceAdapter.kt

```diff
@@ -156,6 +154,29 @@ class InsuranceAdapter(
                             firstStatusPill.show()
                             firstStatusPill.setText(R.string.DASHBOARD_INSURANCE_STATUS_TERMINATED)
                         }
+                        contractStatus.asActiveStatus?.let {
```

Comment:
For correctness, we should probably apply the gray background to the card when it is in an inactive state.